.github/workflows/ci.yml

```diff
@@ -54,12 +54,20 @@ jobs:
           CI: "true"
       - name: Print images
         run: docker images
+      # fixme(amadeus): temporary disable scanning until 2024-09-15 (known vulnerability)
+      - name: Get current date
+        id: date
+        run: |
+          echo "::set-output name=final-date::$(date -d 2024-09-15 +'%s')"
+          echo "::set-output name=current-date::$(date +'%s')"
       - name: Scan image for vulnerabilities
         uses: anchore/scan-action@v3
         with:
           image: "${{ env.DOCKER_BUILD_REPOSITORY }}:${{ matrix.image }}-${{ matrix.architecture }}-${{ env.SHORT_SHA }}"
           severity-cutoff: high
-          fail-build: ${{ matrix.image == 'wolfi-base' || matrix.image == "wolfi-py3.12-slim" }}
+          # fixme(amadeus): temporary disable scanning until 2024-09-15 (known vulnerability)
```

Comment:
I'd probably set a closer date, like ~2 weeks. It's a shame that wolfi has issues with their libreoffice's apk :(

Reply:
Started #39 to address this, hopefully we'll have it fixed this week.